src/test/resources/testbib/bibWithUserComments.bib

```diff
@@ -33,12 +33,12 @@ @Comment this
 @INPROCEEDINGS{1137631,
   author = {Gustav Bostr\"{o}m and Jaana W\"{a}yrynen and Marine Bod\'{e}n and
 	Konstantin Beznosov and Philippe Kruchten},
-  title = {Extending XP practices to support security requirements engineering},
   booktitle = {SESS '06: Proceedings of the 2006 international workshop on Software
 	engineering for secure systems},
+  title = {Extending XP practices to support security requirements engineering},
   year = {2006},
-  pages = {11--18},
```

Comment:
This is also inconsitent to the other file.

Is this file broken maybe? - Can the change be reverted?